src/components/Data/Documents/DocumentListItem.vue

```diff
@@ -121,6 +122,11 @@ export default {
     },
     checkboxId() {
       return `checkbox-${this.document.id}`
+    },
+    formatedDocument () {
```

Comment:
:gb:
```suggestion
    formattedDocument () {
```

Reply:
Also, it would be probably nice to add a comment to explain the motivation of this transform, which might seem weird to a developer that sees this in a year ("da hell is that? KLEAN KODEZ ATTITUDE LOL! Let's delete this")

Reply:
(could perfectly be me, btw)